config/prometheus/clusterrole.yaml

```diff
@@ -11,3 +11,16 @@ rules:
       - "/ols-metrics-access"
     verbs:
       - "get"
+  # Allow access to lightspeed-stack service (required by lightspeed-stack K8s auth module)
+  - nonResourceURLs:
+      - "/ls-access"
+    verbs:
+      - "get"
+  # Allow SubjectAccessReview for authorization checks (required by lightspeed-stack K8s auth module)
+  - apiGroups: ["authorization.k8s.io"]
```

Comment:
prometheus does not need access to these 2 resources for accessing the metrics endpoint:
1. authorization.k8s.io.subjectaccessreviews
2. authentication.k8s.io.tokenreviews

Reply:
fixed